client/src/routes/SignInModal/SignInModal.tsx

```diff
@@ -38,9 +38,6 @@ const SignInModal = () => {
   const signIn = useCallback(async () => {
     try {
       setIsSigningIn(true);
-      if (auth().currentUser) {
-        await auth().signOut();
-      }
```

Comment:
I guess you don't need to sign out before signing in? Or where there other reasons to remove this?

Reply:
Exactly! I was wondering about that, so I tested it and it works without signing out.
What I observed was that if you try to sing in and "it doesn't work" you'll have signed out and not signed in either.